src/storybook/functions/createStoryMetaSettingsDecorator.js

```diff
@@ -0,0 +1,21 @@
+import { iconsMetaData } from "monday-ui-style/src/Icons/iconsMetaData";
+import * as AllIcons from "../../components/Icon/Icons";
+import { createStoryMetaSettings } from "vibe-storybook-components";
+
+export function createStoryMetaSettingsDecorator({
```

Comment:
1. Can you explain why this change was done?
2. Worth adding a comment explaining the purpose of this function.
3. Should it affect the plop as well?

Reply:
1,2. Added a comment there 
`Decorator on createStoryMetaSettings from vibe-storybook-components - adds icons metadata and icons components to function arguments`
3. Plop changes were made already